Homework-Feb-29/BWT/BWT.cs

```diff
@@ -0,0 +1,62 @@
+using System.Text;
+
+namespace BWT;
+
+class BWT
+{
+    const int AlphabetSize = 256;
+
+    public static Tuple<string, int> Transform(string sourceString)
```

Comment:
```suggestion
    public static (string "тут даже название написать можно", int "тут тоже") Transform(string sourceString)
```

По факту тот же Тупл